src/OrchardCore/OrchardCore.Mvc.Core/ModelBinding/ModelError.cs

```diff
@@ -0,0 +1,20 @@
+namespace OrchardCore.Mvc.ModelBinding
+{
+    public class ModelError
+    {
+        public ModelError()
+        {
+
+        }
+
+        public ModelError(string key, string message)
```

Comment:
```suggestion
        public ModelError(string key, string message)
```

Reply:
So the properties should be `readonly`

Reply:
The code suggestion is about removing the empty ctor.